lms/djangoapps/email_marketing/tasks.py

```diff
@@ -184,7 +189,7 @@ def _get_or_create_user_list_for_site(sailthru_client, site=None, default_list_n
     :param: default_list_name
     :return: list name if exists or created else return None
     """
-    if site and site.get('id') != settings.SITE_ID:
+    if not is_white_label(site):
```

Comment:
If `site` is `None`, the next statement will fail and throw exception.
In here, it's OK to check whether the `site` is `None` or not again